reference/kabusapi/REST_API.md

```
@@ -0,0 +1,102 @@
+
+1. `/token` エンドポイントに `POST` して API トークンを取得します。  
+   リクエストボディには YAML に定義されている `RequestToken` スキーマを JSON で渡します。
+2. 以降のリクエストでは、**必ず** HTTP ヘッダー `X-API-KEY` に取得したトークンを設定してください。  
+   (例) `X-API-KEY: a1b2c3d4…`
+3. トークンは以下のタイミングで失効します。
+   - kabu ステーションを終了 / ログアウトしたとき
+   - 新しいトークンを再発行したとき (古いトークンは無効化されます)
+
+## レートリミット
+
+* 発注系: おおよそ **5 req/sec**
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)